asmcli/asmcli

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env bash
```

Comment:
Non-executables should have a .sh extension

Reply:
Done